services/src/main/resources/locale/notification/PostUpdateStatePlugin_fr.properties

```diff
@@ -0,0 +1,20 @@
+UINotification.label.cmsTemplates=Validation de contenu
+UINotification.label.PostUpdateStatePlugin=L'état d'un contenu a été changé
```

Comment:
You need to replace all accented characters with their Unicode representation like : \u00E9 for é

Reply:
@ahamdi done